main.py

```diff
@@ -1,19 +1,35 @@
-import http.server
 import os
 
-PORT = 8000
+from flask import Blueprint, Flask, send_from_directory
 
-web_dir = os.path.join(os.path.dirname(__file__), 'dist')
-os.chdir(web_dir)
+from app import create_app, notify_celery  # noqa
+from app.render import alerts_from_yaml, get_rendered_pages
 
-Handler = http.server.SimpleHTTPRequestHandler
+dev = Blueprint('dev', __name__)
```

Comment:
Optional: we don't actually need a blueprint, especially since we'll only ever have one route. We can just create the app and call `@app.route` directly. Normally if we have a blueprint it would go in a separate file, as a way of splitting up the code.

Reply:
Done in https://github.com/alphagov/notifications-govuk-alerts/compare/27f6191bdf2a4c1fb3c7f99cfc1e3966d29fc5c4..d45fcfe81b45d7573cb42a5bd1ac2b28313ff6c3